gloo/docs/user_guides/gateway/configuring_route_options/advanced_routing_action.md

```diff
@@ -195,11 +203,134 @@ apiVersion: gateway.solo.io/v1
+to the service that should receive traffic and `port` is the port on which the service is listening.
+
+The following configuration will forward all requests to `/petstore` to port `8080` on the Kubernetes service named 
+`petstore` in the `default` namespace.
+
+{{< highlight yaml "hl_lines=6-10" >}}
+routes:
+- matcher:
+    prefix: /petstore
+  routeAction:
+    single:
+      kube:
+        ref:
+          name: petstore
+          namespace: default
+        port: 8080
+{{< /highlight >}}
+
+#### Consul destination {#dest-consul}
+
+Gloo is capable of discovering services registered with [HashiCorp Consul](https://www.hashicorp.com/products/consul/). 
+If this feature has been enabled via the [correspondent field]({{% ref "/v1/github.com/solo-io/gloo/projects/gloo/api/v1/settings.proto.sk#servicediscoveryoptions" %}}) 
+in the `Settings` resource, it is possible to specify Consul services as routing destinations.
+
+A single Consul service usually maps to several service instances, which can have distinct sets of tags, listen on 
+different ports, and live in multiple data centers. To give a concrete example, here is a simplified response you might 
+get when querying Consul for a service with a given name:
+
+```json
+[
+  {
+    "ServiceID": "32a2a47f7992:nodea:5000",
+    "ServiceName": "my-db",
+    "Address": "192.168.1.1",
+    "Datacenter": "dc1",
+    "ServicePort": 5000,
+    "ServiceTags": [
+      "primary"
+    ]
+  },
+  {
+    "ServiceID": "42a2a47f7992:nodeb:5001",
+    "ServiceName": "my-db",
+    "Address": "192.168.1.2",
+    "Datacenter": "dc1",
+    "ServicePort": 5001,
+    "ServiceTags": [
+      "secondary"
+    ]
+  },
+  {
+    "ServiceID": "52a2a47f7992:nodec:6000",
+    "ServiceName": "my-db",
+    "Address": "192.168.2.1",
+    "Datacenter": "dc2",
+    "ServicePort": 6000,
+    "ServiceTags": [
+      "secondary"
+    ]
+  }
+]
+```
+
+The [`consul` destination type]({{% ref "/v1/github.com/solo-io/gloo/projects/gloo/api/v1/proxy.proto.sk#consulservicedestination" %}}) 
+allows you to target a subset of these service instances via the optional `tags` and `dataCenters` fields. Gloo will 
+detect the correspondent IP addresses and ports and load balance traffic between them.
+
+{{% notice note %}}
+When providing the `tags` option, Gloo will only match service instances that **exactly** match the given tag set.
+{{% /notice %}}
+
+For example, the following configuration will forward all matching requests to the second and third service instances,
+
+{{< highlight yaml "hl_lines=6-9" >}}
+routes:
+- matcher:
+    prefix: /db
+  routeAction:
+    single:
+      consul:
+        serviceName: my-db
+        tags:
+        - secondary
+{{< /highlight >}}
+
+while this one will forward the same requests only to the first two instances (the ones in data center `dc1`)
```

Comment:
change to 'while this next example will forward'. hard to tell if 'one' is referring to next or previous example

Reply:
Updated